dgbpy/mlapply.py

```diff
@@ -319,7 +319,9 @@ def doTrain( examplefilenm, platform=dgbkeys.kerasplfnm, type=TrainType.New,
                                         scale=True, force=False,
                                         nbchunks=params['nbchunk'],
                                         split=validation_split )
-    logdir = dgbkeras.getLogDir( examplefilenm, logdir, clearlogs, args )
+    logdir=None
+    if params['withtensorboard']:
```

Comment:
always check first if the key is present in the dictionary:
`if 'withtensorboard' in params and params['withtensorboard']`